scripts/api-proxy.js

```diff
@@ -0,0 +1,45 @@
+// credit: https://github.com/garmeeh/local-cors-proxy/blob/master/lib/index.js
+
+const express = require('express');
+const request = require('request');
+const cors = require('cors');
+const chalk = require('chalk');
+const proxy = express();
+
+var startProxy = function (port, proxyUrl, proxyPartial, credentials, origin) {
+  proxy.use(cors({ credentials: credentials, origin: origin }));
+  proxy.options('*', cors({ credentials: credentials, origin: origin }));
+
+  // remove trailing slash
+  var cleanProxyUrl = proxyUrl.replace(/\/$/, '');
+  // remove all forward slashes
+  var cleanProxyPartial = proxyPartial.replace(/\//g, '');
+
+  proxy.use('/' + cleanProxyPartial, function (req, res) {
+    try {
+      console.log(
+        chalk.green('🏄🏽‍♂️  [API Proxy Plugin]'),
```

Comment:
Btw this is how it looks on my machine:
![image](https://user-images.githubusercontent.com/1494211/84744596-c5c12e00-afb3-11ea-9d4a-573277b511a5.png)

Maybe we should change the surfer to "Person surfing" 🏄 without skin tone for compatibility?

```suggestion
        chalk.green('🏄  [API Proxy Plugin]'),
```